src/modules/UI/components/Buttons/style.js

```diff
@@ -14,7 +14,7 @@ export const styles = {
     height: THEME.BUTTONS.HEIGHT,
     alignItems: 'center',
     justifyContent: 'center',
-    width: '100%',
+    flex: 1,
```

Comment:
This scares me. It would affect every button in the app. How did the create wallet buttons break in the first place?

Reply:
@paullinator This is fixing a *regression* by Allen's onboarding PR that created the exact type of incident that you were scared of. It shouldn't have been adjusted in his PR, and I should have caught it when I code reviewed it. In this PR I have changed the button he is using to the one our app is migrating to, but also changing the styling of the old button back to what it should be (so now the Create Wallet buttons will be back to how their intended style).

I hope this makes sense.